homework5/task1.py

```diff
@@ -0,0 +1,16 @@
+text = input("Enter some text: ")
+char = text.split()
```

Comment:
There is no sense in this line

Reply:
I have already found out it. It can be only input("Enter some text: ").spit. Am I right?